src/panels/config/zwave/zwave-network.ts

```diff
@@ -210,8 +190,15 @@ export class ZwaveNetwork extends LitElement {
   }
 
   private _handleEvent(event) {
-    this._getNetworkStatus();
-    this._networkStarting = event.event_type === "zwave.network_start";
+    if (event.event_type === "zwave.network_start") {
+      // Optimistically set the state, wait 1s and poll the backend
+      // The backend will still report a state of 0 when the 'network_start'
+      // event is first fired.
+      this._networkStatus!["state"] = 5;
```

Comment:
This is not going to cause a re-render because Lit is not observing things inside objects. You'll need to make a copy. 

However, `this._networkStatus` might not exist if an event comes in before network status has been fetched. So I would guard that before setting it. 


```ts
if (this._networkStatus) {
  this._networkStatus = {...this._networkStatus, state: 5};
}
```